$NetBSD: patch-src_fe__utils_Makefile,v 1.2 2022/10/22 18:04:08 nros Exp $

* make sure catalog/pg_type_d.h has been generated before print.c compiles
  fixes build postgresql client when MAKE_JOBS is set to a high value

--- src/fe_utils/Makefile.orig	2022-08-08 20:45:58.000000000 +0000
+++ src/fe_utils/Makefile
@@ -30,6 +30,8 @@ OBJS = \
 	simple_list.o \
 	string_utils.o
 
+print.o: | submake-generated-headers
+
 all: libpgfeutils.a
 
 libpgfeutils.a: $(OBJS)
